src/monali/Assignment_12/Assi_12_Static.java

```diff
@@ -0,0 +1,77 @@
+package monali.Assignment_12;
+
+public class Assi_12_Static {
+	
+	    double num1;
+		double num2;
+        double num3;
+
+
+	  double sum()  {
+		double Answer = num1+num2+num3;
+		return Answer; 
+	}
+	
+	double average1(){
+		double sum1 = sum();
+	    double average =sum1 /3;
+	    return average;
+	}
+	
+	boolean isAnswerAboveExpectation() {
+		double average1 = average1();
+		if(average1 >= 50) {
+		System.out.println("Answer is above Expectation");
+		return true;
+		}
+		else{
+			System.out.println("Answer is below Expectation");
+			return false;
+		}
+	}
+		
+		static boolean isEligible() {
+			Assi_12_Static assi_12_static = new Assi_12_Static();
+			 double average1 = assi_12_static.average1();
+		if(average1 >= 50 || average1 %2 ==0) {
```

Comment:
we need to use divisible function not modular.
if value is more than 50 or divisible by 2. Eligibility will be printed in main method.

Reply:
ok got it sir.